app/scripts/modules/kubernetes/securityGroup/configure/wizard/upsert.controller.js

```diff
@@ -181,6 +183,14 @@ module.exports = angular.module('spinnaker.securityGroup.kubernetes.create.contr
             cloudProvider: 'kubernetes',
             region: $scope.securityGroup.namespace,
           };
+
+          // Turn it into an array so clouddriver can work with it (if it isn't already an array)
```

Comment:
Please make this comment a little more specific here (e.g., the inverse of your comment below - `Change TLS hosts from string to array for Clouddriver`).